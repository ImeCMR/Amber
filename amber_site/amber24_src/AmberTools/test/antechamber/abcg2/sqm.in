Run semi-empirical minimization
 &qmmm
    qm_theory='AM1', grms_tol=0.0005,
 scfconv=1.d-10, ndiis_attempts=700,   qmcharge=1,
 /
   6    C1        3.5400        1.4200        0.0000 
   1    H1        2.5460        1.2490       -0.3830 
   1    H2        4.2190        0.6730       -0.3830 
   1    H3        3.5320        1.3950        1.0800 
   7    N1        4.0060        2.7710       -0.4390 
   1    H4        4.0060        2.7710       -1.4480 
   6    C2        3.0680        3.8500        0.0010 
   1    H5        3.4180        4.7960       -0.3830 
   1    H6        2.0830        3.6360       -0.3830 
   1    H7        3.0510        3.8700        1.0800 
   6    C3        5.4100        3.0440        0.0010 
   1    H8        5.7170        4.0040       -0.3830 
   1    H9        6.0550        2.2680       -0.3830 
   1   H10        5.4360        3.0480        1.0800 

