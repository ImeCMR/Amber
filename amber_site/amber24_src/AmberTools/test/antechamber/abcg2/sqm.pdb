ATOM      1  C1  MOL     1       3.546   1.434  -0.001  1.00  0.00           C
ATOM      2  H1  MOL     1       2.509   1.244  -0.389  1.00  0.00           H
ATOM      3  H2  MOL     1       4.245   0.645  -0.389  1.00  0.00           H
ATOM      4  H3  MOL     1       3.536   1.407   1.122  1.00  0.00           H
ATOM      5  N1  MOL     1       4.006   2.771  -0.463  1.00  0.00           N
ATOM      6  H4  MOL     1       4.006   2.770  -1.493  1.00  0.00           H
ATOM      7  C2  MOL     1       3.078   3.839  -0.003  1.00  0.00           C
ATOM      8  H5  MOL     1       3.432   4.831  -0.392  1.00  0.00           H
ATOM      9  H6  MOL     1       2.046   3.627  -0.391  1.00  0.00           H
ATOM     10  H7  MOL     1       3.059   3.861   1.120  1.00  0.00           H
ATOM     11  C3  MOL     1       5.395   3.041  -0.002  1.00  0.00           C
ATOM     12  H8  MOL     1       5.728   4.040  -0.392  1.00  0.00           H
ATOM     13  H9  MOL     1       6.078   2.237  -0.390  1.00  0.00           H
ATOM     14  H10 MOL     1       5.423   3.047   1.120  1.00  0.00           H
