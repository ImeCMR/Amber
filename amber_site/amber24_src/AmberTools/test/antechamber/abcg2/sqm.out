            --------------------------------------------------------
                             AMBER SQM VERSION 14
 
                                     By
              Ross C. Walker, Michael F. Crowley, Scott Brozell,
                         Tim Giese, Andreas W. Goetz,
                        Tai-Sung Lee and David A. Case
 
            --------------------------------------------------------
 

--------------------------------------------------------------------------------
  QM CALCULATION INFO
--------------------------------------------------------------------------------

| QMMM: Citation for AMBER QMMM Run:
| QMMM: R.C. Walker, M.F. Crowley and D.A. Case, J. COMP. CHEM. 29:1019, 2008

QMMM: SINGLET STATE CALCULATION
QMMM: RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS = 13

| QMMM: *** Selected Hamiltonian *** 
| QMMM: AM1         

| QMMM: *** Parameter sets in use ***
| QMMM: C : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)
| QMMM: H : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)
| QMMM: N : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)

| QMMM: *** SCF convergence criteria ***
| QMMM: Energy change                :  0.1D-09 kcal/mol
| QMMM: Error matrix |FP-PF|         :  0.1D+00 au
| QMMM: Density matrix change        :  0.5D-06
| QMMM: Maximum number of SCF cycles :     1000

| QMMM: *** Diagonalization Routine Information ***
| QMMM: Pseudo diagonalizations are allowed.
| QMMM: Auto diagonalization routine selection is in use.
| QMMM:
| QMMM: Timing diagonalization routines:
| QMMM:                              norbs =       26
| QMMM:    diag iterations used for timing =       20
| QMMM:
| QMMM:              Internal diag routine = 0.002936 seconds
| QMMM:                 Dspev diag routine = 0.005214 seconds
| QMMM:                Dspevd diag routine = 0.004429 seconds
| QMMM:                Dspevx diag routine = 0.015264 seconds
| QMMM:                 Dsyev diag routine = 0.004628 seconds
| QMMM:                Dsyevd diag routine = 0.003740 seconds
| QMMM:                Dsyevr diag routine = 0.003675 seconds
| QMMM:
| QMMM:                Pseudo diag routine = 0.000719 seconds
| QMMM:
| QMMM: Using internal diagonalization routine (diag_routine=1).

  QMMM: QM Region Cartesian Coordinates (*=link atom) 
  QMMM: QM_NO.   MM_NO.  ATOM         X         Y         Z
  QMMM:     1        1      C        3.5400    1.4200    0.0000
  QMMM:     2        2      H        2.5460    1.2490   -0.3830
  QMMM:     3        3      H        4.2190    0.6730   -0.3830
  QMMM:     4        4      H        3.5320    1.3950    1.0800
  QMMM:     5        5      N        4.0060    2.7710   -0.4390
  QMMM:     6        6      H        4.0060    2.7710   -1.4480
  QMMM:     7        7      C        3.0680    3.8500    0.0010
  QMMM:     8        8      H        3.4180    4.7960   -0.3830
  QMMM:     9        9      H        2.0830    3.6360   -0.3830
  QMMM:    10       10      H        3.0510    3.8700    1.0800
  QMMM:    11       11      C        5.4100    3.0440    0.0010
  QMMM:    12       12      H        5.7170    4.0040   -0.3830
  QMMM:    13       13      H        6.0550    2.2680   -0.3830
  QMMM:    14       14      H        5.4360    3.0480    1.0800

--------------------------------------------------------------------------------
  RESULTS
--------------------------------------------------------------------------------

      iter         sqm energy              rms gradient
      ----    -------------------    -----------------------
xmin    10      151.8954 kcal/mol        0.0058 kcal/(mol*A)
  ... geometry converged !
 
 Final MO eigenvalues (eV):
 -45.6739  -34.0996  -34.0996  -29.9914  -24.5021  -22.3491  -22.3491  -18.9851
 -18.9851  -18.4014  -18.3879  -18.3878  -18.1784   -4.2950   -2.1938   -2.1860
  -2.1860   -1.1868   -1.1868   -1.0552   -0.9374   -0.9374   -0.7561   -0.6909
  -0.6113   -0.6113

 Heat of formation   =        151.89538553 kcal/mol  (        6.58667818 eV)

 Total SCF energy    =     -16649.04868256 kcal/mol  (     -721.95692652 eV)
 Electronic energy   =     -63734.52391797 kcal/mol  (    -2763.73634786 eV)
 Core-core repulsion =      47085.47523541 kcal/mol  (     2041.77942134 eV)
 
    Atomic Charges for Step       1 :
  Atom    Element       Mulliken Charge
     1      C                 -0.196
     2      H                  0.148
     3      H                  0.148
     4      H                  0.149
     5      N                  0.003
     6      H                  0.251
     7      C                 -0.196
     8      H                  0.148
     9      H                  0.148
    10      H                  0.149
    11      C                 -0.196
    12      H                  0.148
    13      H                  0.148
    14      H                  0.149
 Total Mulliken Charge =       1.000
 
                  X        Y        Z     TOTAL  
  QM DIPOLE    19.243   13.309   -1.416   23.440
 
 Final Structure

  QMMM: QM Region Cartesian Coordinates (*=link atom) 
  QMMM: QM_NO.   MM_NO.  ATOM         X         Y         Z
  QMMM:     1        1      C        3.5455    1.4338   -0.0010
  QMMM:     2        2      H        2.5087    1.2436   -0.3892
  QMMM:     3        3      H        4.2450    0.6453   -0.3892
  QMMM:     4        4      H        3.5363    1.4071    1.1216
  QMMM:     5        5      N        4.0062    2.7708   -0.4629
  QMMM:     6        6      H        4.0062    2.7701   -1.4934
  QMMM:     7        7      C        3.0784    3.8387   -0.0026
  QMMM:     8        8      H        3.4324    4.8314   -0.3916
  QMMM:     9        9      H        2.0460    3.6268   -0.3913
  QMMM:    10       10      H        3.0592    3.8612    1.1199
  QMMM:    11       11      C        5.3947    3.0408   -0.0021
  QMMM:    12       12      H        5.7276    4.0404   -0.3917
  QMMM:    13       13      H        6.0776    2.2375   -0.3898
  QMMM:    14       14      H        5.4231    3.0475    1.1204

           --------- Calculation Completed ----------

